bots/images/scripts/debian.setup

```diff
@@ -23,6 +23,7 @@ libpwquality-tools \
 libssh-4 \
 libteam-utils \
 libvirt-daemon-system \
+libvirt-dbus \
```

Comment:
debian.setup is being shared between all debian and ubuntu flavors. As libvirt-dbus isn't in debian-stable yet, nor in any released Ubuntu yet, this needs to be restricted to the debian-testing image. Look around line 60, there is a `case` switch for special-cases for the images. Please add a `testing)` case and add libvirt-dbus there: `COCKPIT_DEPS="${COCKPIT_DEPS} libvirt-dbus"`